docs/ModuleWidgets.md

```diff
@@ -0,0 +1,39 @@
+## Module Widgets
+
+Since LORIS 23.0, LORIS incorporates a concept of "widgets" which
+modules can use to dynamically display graphical widgets in other
+modules when installed and activated. This is a powerful way to
+extend the capabilities of other modules within a new module.
+
+Widgets are can be used by implementing the `getWidgets` method
+on a `Module` module descriptor class. The first parameter of
+`getWidgets` is a type string, allowing modules to define their
+own widget types in a loosely coupled manner. However since the
+interface is abstract enough to be flexible there must be an implied
+contract between what options and their returned values that are not
+enforced by PHP's typing system.
+
+Widget types currently implemented by LORIS and its default modules are
```

Comment:
Incomplete sentence? If this is meant to lead into the list it should probably end with a colon or should be removed.

It is worth considering whether the following info should be moved into the modules' respective READMEs instead of this document. As more widgets are added, I think it would be better and more realistic that devs update a module's README with widget information rather than come back to this document

Reply:
The document is intended to be a centralized place to figure out what you can do using the widget system when writing a new module.

I added the missing colon.

Reply:
In that case maybe this should be added to #6048 instead? Widgets could be subsection fo that document.

Reply:
I have nothing against incorporating it into a `Creating a Module` guide in the future, but I think this should be documented in this release, since this is the release where widgets were added.